### DISCUSSION TITLE
RichTextEditorHtml updating Raw HTML Editor but not Rich Text Editor

### DISCUSSION THREAD

**Discussion by artmedia** [oqtane/oqtane.framework#3135] [General]
Hello,
I'm using Oqtane RichTextEditorHtml in my custom module to create/edit news.
To edit news I open a modal in which there is RichTextEditorHtml.
When I open the modal for the first news, both Rich Text Editor and Raw HTML Editor are populated correctly.
When I close modal and reopen it to edit another news,  only Raw HTML Editor is populated correctly, 
Rich Text Editor is not refreshed and it still contains the previous news content.
 
My code is this:
private async Task openModalEdit(News news)
{
        var interop = new Artmedia.AlbalaghaNews.Interop(JSRuntime);
        await interop.OpenModal("modalInsertEditNews");	
        _content = await NewsContentService.Get(ModuleState.ModuleId, news.IdNews);
	this.StateHasChanged();
}

Anyone knows how to make Rich Text Editor to refresh every time?

Thanks
Luca

**Comment by leigh-pointer** [comment]
@artmedia I would suggest taking a look at the HTML module in the Oqtane.Client project.

**Reply by artmedia** [reply]
yes, it's the same use that I see in HTML module.
The only difference is that I open it twice in the same module, HTML module is opened always as new instance

**Reply by leigh-pointer** [reply]
@artmedia Make sure the id / names are unique for the jscript interop
